src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+
+import java.util.Scanner;
+
+class Task {
+    String taskName;
+    boolean status;
+
+    Task() {
+        this.status = false;
+    }
+
+    public String getTaskStatus() {
+        String status = " ";
+
+        if (this.status == true) {
+            status = "X";
+        }
+        return status;
+    }
+
+    public void mark() {
+        this.status = true;
+        System.out.println("目標達成！- [" + this.getTaskStatus() + "] " + this.taskName);
+
+    }
+
+    public void unMark() {
+        this.status = false;
+        System.out.println("えええ？噓つき。- [" + this.getTaskStatus() + "] " + this.taskName);
+    }
+}
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] taskList = new Task[100];
```

Comment:
might be worth defining 100 as a variable to avoid magic number (maybe maxNumTasks)